lib/location/method_gnss.c

```diff
@@ -271,6 +278,7 @@ static void method_gnss_nrf_cloud_agps_request(void)
 		return;
 	}
 	rest_ctx.auth = (char *)jwt_buf;
+#endif
```

Comment:
I'm gonna stop pointing this out -- just generally being explicit about COAP guards seems like a good idea

Reply:
In this case, the whole function is wrapped in:
```
#elif defined(CONFIG_NRF_CLOUD_REST) || defined(CONFIG_NRF_CLOUD_COAP)
...
#endif
so the `#if defined(CONFIG_NRF_CLOUD_REST) #else...` at 281 is fine.